s32/mcux/devices/S32Z270/S32Z270_device.h

```diff
@@ -1885,4 +1885,849 @@ typedef struct {
+#define SPI_CTAR_CSSCK_SHIFT                       DSPI_CTAR_CSSCK_SHIFT
+#define SPI_CTAR_CSSCK_WIDTH                       DSPI_CTAR_CSSCK_WIDTH
+#define SPI_CTAR_CSSCK(x)                          DSPI_CTAR_CSSCK(x)
+
+#define SPI_CTAR_PBR_MASK                          DSPI_CTAR_PBR_MASK
+#define SPI_CTAR_PBR_SHIFT                         DSPI_CTAR_PBR_SHIFT
+#define SPI_CTAR_PBR_WIDTH                         DSPI_CTAR_PBR_WIDTH
+#define SPI_CTAR_PBR(x)                            DSPI_CTAR_PBR(x)
+
+#define SPI_CTAR_PDT_MASK                          DSPI_CTAR_PDT_MASK
+#define SPI_CTAR_PDT_SHIFT                         DSPI_CTAR_PDT_SHIFT
+#define SPI_CTAR_PDT_WIDTH                         DSPI_CTAR_PDT_WIDTH
+#define SPI_CTAR_PDT(x)                            DSPI_CTAR_PDT(x)
+
+#define SPI_CTAR_PASC_MASK                         DSPI_CTAR_PASC_MASK
+#define SPI_CTAR_PASC_SHIFT                        DSPI_CTAR_PASC_SHIFT
+#define SPI_CTAR_PASC_WIDTH                        DSPI_CTAR_PASC_WIDTH
+#define SPI_CTAR_PASC(x)                           DSPI_CTAR_PASC(x)
+
+#define SPI_CTAR_PCSSCK_MASK                       DSPI_CTAR_PCSSCK_MASK
+#define SPI_CTAR_PCSSCK_SHIFT                      DSPI_CTAR_PCSSCK_SHIFT
+#define SPI_CTAR_PCSSCK_WIDTH                      DSPI_CTAR_PCSSCK_WIDTH
+#define SPI_CTAR_PCSSCK(x)                         DSPI_CTAR_PCSSCK(x)
+
+#define SPI_CTAR_LSBFE_MASK                        DSPI_CTAR_LSBFE_MASK
+#define SPI_CTAR_LSBFE_SHIFT                       DSPI_CTAR_LSBFE_SHIFT
+#define SPI_CTAR_LSBFE_WIDTH                       DSPI_CTAR_LSBFE_WIDTH
+#define SPI_CTAR_LSBFE(x)                          DSPI_CTAR_LSBFE(x)
+
+#define SPI_CTAR_CPHA_MASK                         DSPI_CTAR_CPHA_MASK
+#define SPI_CTAR_CPHA_SHIFT                        DSPI_CTAR_CPHA_SHIFT
+#define SPI_CTAR_CPHA_WIDTH                        DSPI_CTAR_CPHA_WIDTH
+#define SPI_CTAR_CPHA(x)                           DSPI_CTAR_CPHA(x)
+
+#define SPI_CTAR_CPOL_MASK                         DSPI_CTAR_CPOL_MASK
+#define SPI_CTAR_CPOL_SHIFT                        DSPI_CTAR_CPOL_SHIFT
+#define SPI_CTAR_CPOL_WIDTH                        DSPI_CTAR_CPOL_WIDTH
+#define SPI_CTAR_CPOL(x)                           DSPI_CTAR_CPOL(x)
+
+#define SPI_CTAR_FMSZ_MASK                         DSPI_CTAR_FMSZ_MASK
+#define SPI_CTAR_FMSZ_SHIFT                        DSPI_CTAR_FMSZ_SHIFT
+#define SPI_CTAR_FMSZ_WIDTH                        DSPI_CTAR_FMSZ_WIDTH
+#define SPI_CTAR_FMSZ(x)                           DSPI_CTAR_FMSZ(x)
+
+#define SPI_CTAR_DBR_MASK                          DSPI_CTAR_DBR_MASK
+#define SPI_CTAR_DBR_SHIFT                         DSPI_CTAR_DBR_SHIFT
+#define SPI_CTAR_DBR_WIDTH                         DSPI_CTAR_DBR_WIDTH
+#define SPI_CTAR_DBR(x)                            DSPI_CTAR_DBR(x)
+/*! @} */
+
+/*! @name CTAR_SLAVE - Clock and Transfer Attributes Register (In Slave Mode) */
+#define SPI_CTAR_SLAVE_CPHA_MASK                   (0x2000000U)

```

Comment:
I thought RTD already defines all macros we need? isn't it

Reply:
I remove all the RTD unsupported macros